interview-questions.md

```diff
@@ -7,44 +7,57 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: Interestingly, I made this same mistake recently when making the Wildlife Tracker. To resolve this challenge question, I would simply run the following 2 terminal commands: `rails g migration adds_foreign_key_to_students` Afterwards I would run `rails db:migrate` Since Cohort has many students, foreign key in this case gets added to Students. 
 
-  Researched answer:
+  Researched answer: In order to research this question and since this was the exact mistake I made in the Wildlife Tracker, I went back ton research how I handled this issue specificly. It is worth noting, that I believe in this challenege question and based on how i handled Wildlife tracker, i believe inside the migration file, in the change method students should be verified to be plural. Since we are talking about keys, it is also worth noting that the primary key in this example would reside with the Cohort model.
```

Comment:
Awesome answers here!  Just to add to this, both models will have a primary key, however the foreign key is referencing the primary key of the Cohort model.